src/itemlistformaction.cpp

```diff
@@ -752,6 +753,15 @@ bool ItemListFormAction::process_operation(Operation op,
 		}
 	}
 	break;
+	case OP_ENQUEUE:
+		if (!visible_items.empty() && itempos < visible_items.size()) {
+			const auto item = visible_items[itempos].first;
+			return EnqueueItemEnclosure(item, feed, *v, *rsscache);
+		} else {
+			v->get_statusline().show_error(_("No item selected"));
```

Comment:
```suggestion
			v->get_statusline().show_error(_("No item selected!"));
```

That's what other ops say in this situation. Let's be consistent :)